resources/local/config.toml

```diff
@@ -32,7 +32,7 @@ format = "text"
 
 # Path (absolute, or relative to this config.toml) to validator's secret key file used to sign
 # consensus messages.
-secret_key_path = 'secret_key.pem'
+secret_key_path = '../secret_key.pem'
```

Comment:
We also don't need this change.

Reply:
Sorry, this change was made when I was trying to understand WithDir, I shall revert accordingly